app/models/patient.rb

```diff
@@ -1,13 +1,18 @@
 # Object representing core patient information and demographic data.
+
 class Patient
   include Mongoid::Document
   include Mongoid::Timestamps
   include Mongoid::Enum
   include Mongoid::History::Trackable
   include Mongoid::Userstamp
   include StatusHelper
+  include Urgency
+  include Callable
+  include Notetakeable
+  include Searchable
+  include AttributeDisplayable
 
```

Comment:
Maybe a note at the top here stating that these are Concerns (with a link to wtf Concerns are?) and any related functionality should go in them instead of directly in Patient?